bpy_speckle/utils/account_manager.py

```diff
@@ -0,0 +1,10 @@
+from specklepy.api.credentials import get_local_accounts
+
+def get_account_enum_items() -> list[tuple[str, str, str]]:
```

Comment:
Current specklepy does not support built-in types yet. We should use the following instead:
```python
from typing import List, Tuple
def get_account_enum_items() -> List[Tuple[str, str, str]]:
```